taskcluster/kinds/bitrise-trigger/kind.yml

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think we shouldn't call this kind `bitrise-trigger` and instead name it after whatever the workflow / pipeline it is calling. For example, maybe in this case it could be called `bitrise-build` or whatever. The main reason is that in Taskgraph, you can't have tasks within the same kind depend on one another. You can only add dependencies on other kinds.

So if we want to have a `publish` trigger at some point, it needs to be defined in a separate `kind.yml` as the `build` trigger (even if the task structure is otherwise the same).

Reply:
Sure, makes sense. I will rename this. Thanks!